### PR TITLE
Adjusted spacing and layout for tables

### PR DESCRIPTION
**Pull request by jakeatdocforce**

### CODE REVIEW

**Comment by mDuo13** [comment]
I'm not sure where the change that causes this is, but this PR appears to make certain tables flow into the right sidebar or even off the right side of the page at some screen widths. Compare the table in `ledger-header.html` on production, staging, and this branch at 1000px wide.

The table is also broken for all three layouts at most mobile screen widths, so that's something we maybe should also fix, but since that's not a regression I'm not as strict about it.

Also, there are some font size and line-height definitions in the existing `devportal.css` definition that should probably be dropped or adjusted.

File: assets/css/devportal.css [hunk 1]
```diff
@@ -274,13 +281,14 @@ body {
 
 .content table {
   clear: right;
-  margin-bottom: 1rem;
+  margin-bottom: 48px;
   font-size: 0.95rem;
```
Comment (mDuo13) [inline assets/css/devportal.css]: You can remove this font-size declaration, and the `line-height: 1.65` declaration on lines 272-274.

File: assets/css/devportal.css [hunk 2]
```diff
@@ -274,13 +281,14 @@ body {
...
 }
 
 .content table code {
   word-break: normal;
   white-space: nowrap;
+  font-size: 1rem
```
Comment (mDuo13) [inline assets/css/devportal.css]: I think it's OK to remove this font size definition as well and let code font in tables use the (bootstrap-default) 87.5% rate. 

Code fonts tend to be bigger than the equivalent body font at the same "pixel" size, so leaving those as bootstrap-default allows them to be consistently sized both in and out of tables. If we want to adjust the bootstrap setting for the code font size across the whole site, we can set the `$code-font-size` variable in the scss file.

It's a little hard to glean the intentions from the mockup, but I think we probably want to leave font sizes  (both monospaced and not) within tables consistent with the font sizes in paragraphs and list items.

File: assets/css/devportal.css [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
   padding: 0.2em;
   vertical-align: text-top;
+  padding: 12px;
 }
 th {
-  padding: 0.2em;
+  border-bottom: 1px solid black;
+}
+tr {
+  border-bottom: 1px solid #ccc;
```
Comment (mDuo13) [inline assets/css/devportal.css]: We should move this color to the `xrpl.scss` file and change it to use one of the grays defined there, probably `$gray-200` to match with the other borders on the tree navs and main content blocks.

Comment (jakeatdocforce) [reply assets/css/devportal.css]: done. changed to black "1b1818"  and gray-200 to match others

File: assets/css/devportal.css (hunk 2 above)
Comment (jakeatdocforce) [reply assets/css/devportal.css]: removed.  Text is slightly smaller but still looks good

File: assets/css/devportal.css (hunk 1 above)
Comment (jakeatdocforce) [reply assets/css/devportal.css]: Removed.  
I think table overflow might be part of the padding being added and some long words pushing it out.   "close_time_resolution"  for example wont break anywhere to allow that col to be smaller.

**Pull request merged**